tests/console/apache.pm

```diff
@@ -46,6 +46,20 @@ sub run {
     systemctl 'start apache2';
     systemctl 'status apache2';
 
+    # Check if apache is working when php module is enabled
+    if (is_sle('<=12-SP5')) {
```

Comment:
It'd be maybe nice to include also older versions, or?

Reply:
> It'd be maybe nice to include also older versions, or?

Hmmm what do you mean? ^^

Reply:
You do `if (is_sle('<=12-SP5'))` but there is no `else` so I wonder how this behaves on other versions.

Reply:
Comments are updated :) For SLE15+, dependencies are already installed and php module is enabled properly, so nothing changes in the terms of testing!